itools-packager/src/main/resources/itools

```diff
@@ -41,11 +41,11 @@ for ((i=0; i<${#args[@]}; i++)); do
 done
 
 [ -n "$itools_cache_dir" ] && options+="-Ditools.cache.dir=$itools_cache_dir"
-[ -n "$itools_config_dir" ] || itools_config_dir="${HOME}/.itools"
-options+=" -Ditools.config.dir=$itools_config_dir"
-[ -n "$itools_config_name" ] && options+=" -Ditools.config.name=$itools_config_name"
+[ -n "$powsybl_config_dirs" ] || powsybl_config_dirs="${HOME}/.itools"
+options+=" -Dpowsybl.config.dirs=$powsybl_config_dirs"
```

Comment:
The default value of powsybl_config_dirs should be: `$installDir/etc:$HOME/.itools`